.github/workflows/security.yaml

```diff
@@ -1,46 +0,0 @@
----
-
-# source: https://dev.to/aws-builders/improving-your-cicd-pipeline-helm-charts-security-scanning-with-trivy-and-github-actions-3315
-
-name: Security scans
```

Comment:
You don't care about the image scan anymore?

Reply:
It's been moved inside the helm chart pipeline